sample-app/src/components/cards/StandardCard.tsx

```diff
@@ -14,28 +47,62 @@ export interface StandardCardProps extends CardProps {
  * @param props - An object containing the result itself.
  */
 export function StandardCard(props: StandardCardProps): JSX.Element {
-  const { configuration, result } = props;
+  const { configuration, result, customCssClasses, compositionMethod } = props;
+  const cssClasses = useComposedCssClasses(builtInCssClasses, customCssClasses, compositionMethod);
 
-  return (
-    <div className='StandardCard'>
-      {configuration.showOrdinal && result.index && renderOrdinal(result.index)}
-      <div className='StandardCard__contentWrapper'>
-        {result.name && renderTitle(result.name)}
+  const cta1 = result.rawData.c_primaryCTA as CtaData;
+  const cta2 = result.rawData.c_secondaryCTA as CtaData;
+
+  // TODO (cea2aj) We need to handle the various linkType so these CTAs are clickable
+  function renderCTAs(cta1?: CtaData, cta2?: CtaData) {
+    return (<>
+      {(cta1 ?? cta2) && 
+        <div className={cssClasses.ctaContainer}>
+          {cta1 && renderFirstCTA(cta1)}
+          {cta2 && renderSecondCTA(cta2)}
+        </div>
+      }
+    </>);
+  }
+
+  function renderFirstCTA({ label, link, linkType }: CtaData) {
+    return (
+      <button className={cssClasses.cta1}>{label}</button>
+    )
+  }
+
+  function renderSecondCTA({ label, link, linkType }: CtaData) {
+    return (
+      <button className={cssClasses.cta2}>{label}</button>
+    )
+  }
+
+  function renderOrdinal(ordinal: number) {
+    return (
+      <div className={cssClasses.ordinal}>
+        {ordinal} -
```

Comment:
Seems a bit odd with the dash as part of the renderOrdinal's div. Can we have it outside of it? after this renderOrdinal is called

Reply:
We don't have mocks for an ordinal, so is it okay if I just leave it for now?  (Or I could remove the dash). We would will probably update it anyway once we get guidance from UX

Reply:
gotcha, yeah removing it as fine too

Reply:
UX says we can simply not render the ordinal for now, and we'll update once we get mocks